app/serializers/user_serializer.rb

```diff
@@ -8,4 +8,22 @@ def salary
   def photo
     self.object.photo
   end
+
+  def pending_vacations
+    vacations = self.object.vacations.where(status: "pending")
```

Comment:
It looks these models should leave in different place. Model - is a better place for now. It also can be `decorator` but we don't have any atm.